genie-common/src/test/java/com/netflix/genie/common/dto/JobUnitTests.java

```diff
@@ -290,7 +292,7 @@ public void canBuildJobWithNulls() {
         builder.withCommandName(null);
         builder.withFinished(null);
         builder.withStarted(null);
-        builder.withStatus(null);
```

Comment:
Status is no longer optional?

Reply:
It should never have been. It's even marked required in the docs: https://netflix.github.io/genie/docs/3.3.2/rest/#_job